sdk/textanalytics/Azure.AI.TextAnalytics/src/LinkedEntityMatch.cs

```diff
@@ -13,14 +13,18 @@ namespace Azure.AI.TextAnalytics
     [CodeGenModel("Match")]
     public readonly partial struct LinkedEntityMatch
     {
+        // use for deserialization
         internal LinkedEntityMatch(double confidenceScore, string text, int offset, int length)
+            : this(confidenceScore, text, offset) => Length = length;
+
+        internal LinkedEntityMatch(double confidenceScore, string text, int offset)
         {
             // We shipped TA 5.0.0 Text == string.Empty if the service returned a null value for Text.
             // Because we don't want to introduce a breaking change, we are transforming that null to string.Empty
             Text = text ?? string.Empty;
             ConfidenceScore = confidenceScore;
             Offset = offset;
-            Length = length;
+            Length = default;
```

Comment:
should it be removed rather then keep it and having a private getter method?

Reply:
I am reusing the class that codegen is creating from the swagger for LinkedEntityMatch. I could do:
1) Create a new class and copy everything from the one codegen created and remove Length
2) Hide the Length property from public API and Mock methods 

I went with 2. as it involves less copy paste and easier to maintain. Let me know if you feel strongly about going with 1 instead.

Reply:
Noop. I prefer to have less codegene classes. Recently, java codegene is able to do more things, like changes the modifier, name of class, return types. So in the future, I would need to reduced the codegene classes as well. Thanks Mari for the explaination.

Reply:
cool!!! Yeah in .NET we can rename classes, methods, properties and a bunch of things. but removing properties is kind of tricky, so it is easier to leave them as private and let them be :| haha
Thanks Shawn